service/server.py

```diff
@@ -78,7 +78,7 @@ def signin_page():
 
 BAD_LOGIN_COUNTER = Counter()
 MAX_LOGIN_ATTEMPTS = 10
-NOF_SECS_BETWEEN_LOGINS = 2
+NOF_SECS_BETWEEN_LOGINS = 1
```

Comment:
Not very important, but delays are going to be rather small, perhaps it would be best it the unit was milliseconds.